### ISSUE TITLE
Card Component for Enhanced Customization in OnceUI

### ISSUE THREAD

**Issue by SnowDingo** [once-ui-system/nextjs-starter#4]
**Describe the feature**
The feature allows users to access the card component.  The component would support properties like `title`, `description`, `border-radius`,  `background`, and more, allowing for deeper customization.

The card design could be similar to the following examples↓
<img width="1068" alt="スクリーンショット 2024-10-25 17 05 57" src="https://github.com/user-attachments/assets/05db3766-e73e-45da-9bd8-e568db66526e">

<img width="981" alt="スクリーンショット 2024-10-25 17 07 36" src="https://github.com/user-attachments/assets/bc523e36-8da9-4bce-b04c-2de14c0cbff1">

## Suggested properties
* Background: Selection of colors to match with OnceUI’s design system.
* Card Title: A customizable title for the card component.
* Card Description: An optional description for additional detail.
* padding: For inner padding
* Buttons: For quick actions

# Relation to existing components
This request is for a new component, so it doesn't have any relation with existing components.

**Comment by zsofiakm** [comment]
Hi @SnowDingo,
Thanks for this request, and for putting so much thought into it. We have a release coming up this week and could add this to the following one.

If you need this in the meantime, you could use Flex and create a local component.

**Comment by lorant-one** [comment]
@HardikRathod123 yes of course! please [join our Discord server](https://discord.com/invite/5EyAQ4eNdS) if you haven't yet for a smooth collaboration :) we're already working on a component creation guide that outlines the general requirements and standards to keep consistency within the system.

**Comment by lorant-one** [comment]
component added in v0.5
